### DISCUSSION TITLE
JoomCCK v4.0 Alpha - Help for testing

### DISCUSSION THREAD

**Discussion by andiamine** [JoomCoder-com/JoomCCK#36] [Announcements]
First release of JoomCCK for Joomla 4 , this version for testing purpose don't use on production.

### What's new ?
- Joomla 4 support
- Bootstrap 5 support
- PHP 8.1 support

Please everyone help us in testing

**Comment by andiamine** [comment]
Please we need help in testing.

**Comment by Giorgi625** [comment]
I just came home and see this great news. I get ready and start testing right away.

**Reply by andiamine** [reply]
thanks for your help , you are welcome.

**Comment by LordHelmchen666** [comment]
I will test it the next  few days.
Will it be possible to easy upgrade from J3 Cobalt to J4 JoomCCK? Or do we have to migrate?

**Reply by andiamine** [reply]
Yes , i will write an article about it soon.

**Comment by Giorgi625** [comment]
in admin on page JoomCCK Configuration I see language constantes like this XML_FSETLBL_CNF_GLOBAL instead of their values. of default language is set to English.

**Reply by andiamine** [reply]
please open issue in issues https://github.com/JoomCoder-com/JoomCCK/issues

**Comment by andiamine** [comment]
Alpha 3 is available , fix issues on installation and global config.

**Comment by andiamine** [comment]
Alpha 4 is available  :)
https://github.com/JoomCoder-com/JoomCCK/releases

**Comment by andiamine** [comment]
new alpha 5 version available
https://github.com/JoomCoder-com/JoomCCK/releases/tag/4.0a5

Please we need more help in detecting and reporting issues

Thanks to @Giorgi625 & @LordHelmchen666 for help :)
